ui/portal/app/src/index.ts

```diff
@@ -1,9 +1,10 @@
 import "../assets/global.css";
 
-import { install } from "react-native-quick-crypto";
```

Comment:
'react-native-quick-crypto' is a shim and have to be loaded as the first thing in the app.